.devops/local-templates/setup-env.yaml

```diff
@@ -12,13 +12,18 @@ parameters:
     type: string
     default: ''
 
+  - name: 'client_id'
+    type: string
+    default: ''
+
 steps:
 - bash: |
     chmod +x env.sh
     env -i \
       RTP_FE_ENV=${{ parameters.env }} \
       RTP_FE_VERSION=${{ parameters.version }} \
       RTP_FE_API_URL=${{ parameters.api_url }} \
+      RTP_FE_CLIENT_ID=${{ parameters.client_id }} \
```

Comment:
parameter name must match the one in deploy pipeline. at the moment `client_id` here and `vite_auth_client_id` there

Reply:
I'have amended my commit addressing all the referenced variables